.github/workflows/build-and-check.yml

```diff
@@ -99,8 +99,10 @@ jobs:
       - name: set up JDK 1.8
         uses: actions/setup-java@v2
         with:
-          java-version: 8
+          java-version: 11
           distribution: 'zulu'
+      - name: Install lintian
+        run: sudo apt install -qq lintian=2.114.0ubuntu1.3
```

Comment:
checkbashisms was resolving its transitive dependency on lintian to a version that no longer exists. i fixed it by explicitly installing a version of the dependency that actually exists